test_bootcamp_calculator.py

```diff
@@ -47,18 +47,14 @@ def test_divide_two_positive_int(self):
         self.assertEqual(3, self.calculator.divide(3, 1), msg="Wrong calculation result")
 
     def test_divide_two_positive_int_and_float(self):
-        self.assertEqual(1.0, self.calculator.divide(3, 3.0), msg="Wrong calculation result")
+        self.assertEqual(0.449, self.calculator.divide(3.14, 7), msg="Wrong calculation result")
 
     def test_divide_two_negative_int(self):
         self.assertEqual(1, self.calculator.divide(-2, -2), msg="Wrong calculation result")
 
     def test_divide_two_negative_int_and_float(self):
         self.assertEqual(2.0, self.calculator.divide(-4, -2.0), msg="Wrong calculation result")
 
-    def test_divide_by_zero(self):
-        with self.assertRaises(ZeroDivisionError):
-            self.calculator.divide(3, 0)
-
     def test_add_empty_input(self):
```

Comment:
Why have you removed test_divide_by_zero() test? You still need to test that your validation works

Reply:
When I add that new condition (if statement) for dividing by zero, that for that test I received "AssertionError: ZeroDivisionError not raised", because there is no possibility to divide by zero.

Reply:
Please add test to check that your method divide() have correct behavior for such input divide(x=3, y=0)
Think what the output you expect, add check this expected output in your test